src/platform/monitoring/DowntimeNotification/util/helpers.js

```diff
@@ -100,3 +102,15 @@ export function getSoonestDowntime(serviceMap, serviceNames) {
         : service;
     }, null);
 }
+
+export const isGlobalDowntimeInProgress = (
+  downtimeWindow = scheduledDowntimeWindow,
```

Comment:
The only reason this takes a parameter is for testing, right? Is there a reason to pass it a parameter for realz?

Reply:
Yup, good ol' injection.